### PR TITLE
Updating the CHANGELOG

### PR DESCRIPTION
**Pull request by tobiasd**
This should be merged last before the tagging of the 3.5.2.

It's work in progress, currently the closed issues and work on the addons is included in the updated CHANGELOG file.

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,8 +1,41 @@
 Version 3.5.2 (2017-05-XX)
   Friendica Core:
-    
+    Updates to the translations (DE, EN-GB, ES, IT, RU) [translation teams]
+    Updates to the documentation [annando, beardyunixer, rabuzarus, tobiasd]
+    Updated the nginx example configuration [beardyunixer]
+    Code revision and refractoring [annando, hypolite, Quix0r, rebeka-catalina]
```
Comment (annando) [inline CHANGELOG]: Isn't it called "refactoring"?

File: CHANGELOG [hunk 2]
```diff
@@ -1,8 +1,41 @@
...
+    Improved the installer [heluecht]
+    Improved openid handling [strk]
+    Improved php7 support [heluecht]
+    Improved display of notifications [heluecht]
+    Improved logging mechanism [beardyunixer]
+    Behaviour clarification of the group filter / new tab [heluecht]
+    Old options for the pager and share element were removed [heluecht]
+    Support of PDO was added [heluecht]
+    Improved error logging for issues with the database [heluecht]
+    Adoption to changes in DateTime handling of MySQL [heluecht]
```
Comment (annando) [inline CHANGELOG]: You could add some "Improved compatibility for MySQL version ..."

Comment (tobiasd) [inline CHANGELOG]: Which version was it`5.7+ right?

File: CHANGELOG (hunk 1 above)
Comment (tobiasd) [inline CHANGELOG]: Yeah, refractors are way too cool ;-) I'll fix the typo.

File: CHANGELOG (hunk 2 above)
Comment (annando) [inline CHANGELOG]: I guess so.

Comment (tobiasd) [inline CHANGELOG]: done

**Comment by tobiasd** [comment]
@Hypolite it's annando now in the entire document

**Pull request merged**